x-pack/metricbeat/module/awsfargate/cloudformation.yml

```diff
@@ -0,0 +1,82 @@
+AWSTemplateFormatVersion: "2010-09-09"
```

Comment:
Is this valid?

Reply:
Yes I got it from https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/format-version-structure.html :)

Reply:
looks like ancient history but ok! 🤷🏼‍♂️